src/crimson/os/seastore/seastore.cc

```diff
@@ -704,7 +704,9 @@ SeaStore::_omap_get_value_ret SeaStore::_omap_get_value(
       return manager.omap_get_value(root, t, key
       ).si_then([](auto opt) -> _omap_get_value_ret {
         if (!opt) {
-          return crimson::ct_error::enodata::make();
+          bufferlist bl;
+          bl.append("");
+          return seastar::make_ready_future<ceph::bufferlist>(std::move(bl));
```

Comment:
Are we sure this is the correct behavior?  BlueStore::getattr() returns -ENODATA if the attr isn't present.

Reply:
Seems like we should handle ct_error::enodata in PGBackend::cmp_xattr.

Reply:
@athanatos updated in PGBackend::cmp_xattr. please review.